content/en/platform/corda/5.0/deploying-operating/external-messaging/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+date: '2023-06-28'
+title: "External Messaging Administration"
+project: corda
+version: 'Corda 5.0 Beta 4'
+menu:
+  corda5:
+    identifier: corda5-cluster-external-messaging
+    parent: corda5-cluster
+    weight: 10000
+section_menu: corda5
+---
+
+# External Messaging Administration
+
+CorDapp flows can send simple messages via Kafka to external systems. For more information, see [External Messaging CorDapps]({{< relref "../../developing-applications/external-messaging.md" >}}). This section describes the cluster adminstration tasks required for CorDapps that implement external messaging. It contains the following:
```

Comment:
missing the second 'i' in **administration**

Reply:
```suggestion
CorDapp flows can send simple messages via Kafka to external systems. For more information, see [External Messaging CorDapps]({{< relref "../../developing-applications/external-messaging.md" >}}). This section describes the cluster administration tasks required for CorDapps that implement external messaging. It contains the following:
```